homeassistant/components/alexa/capabilities.py

```diff
@@ -1325,6 +1548,8 @@ class AlexaEventDetectionSensor(AlexaCapability):
     https://developer.amazon.com/docs/device-apis/alexa-eventdetectionsensor.html
     """
 
+    supported_locales = "en-US"
```

Comment:
We should make this value either strings or tuples. Note that to make it a tuple with 1 item, you need to include the extra comma.

Reply:
It might also be better to make it a set, for faster lookup? 
```suggestion
    supported_locales = {"en-US"}
```

Reply:
Thanks! I always appreciate the advice. 

I knew tuples were faster than lists in this situation, but your comment made me _"go down the rabbit hole"_ and discover a set would be best here. 

As far as the single item tuple as a string... let's just say I have since disabled the inspection that  lead to that.